### PR TITLE
Add documentation on CF API concurrency rate limit

### PR DESCRIPTION
**Pull request by FloThinksPi**

### CODE REVIEW

**Comment by FloThinksPi** [comment]
CLA Seems to be broken ? 
image

**Comment by je-hal** [comment]
@FloThinksPi : It seems if you wait a little longer, signing the CLA should work, see cla-assistant/cla-assistant#874. Could you please try again?

File: docs/10-concepts/cloud-foundry-environment-9c7092c.md [hunk 1]
```diff
@@ -99,6 +99,8 @@ The following technical configurations are specific to SAP BTP and differ from t
 
 -   On SAP BTP, the Cloud Foundry API is protected by a rate limit against misuse. The limit is in the range of a few 10k requests per hour per user.
 
+-   In addition to the general rate limit on the Cloud Foundry API, requests for certain API endpoints related to services face a separate limit on concurrent requests (see details [here](https://docs.cloudfoundry.org/running/rate-limit-cloud-controller-api.html#Rate%20Limit%20Responses:%20Service%20Brokers)). The CF API responds with http status code 429 if a rate limit is reached and provides a Retry-After header suggesting when the client should attempt a retry.
```
Comment (sdanter) [inline docs/10-concepts/cloud-foundry-environment-9c7092c.md:102]:
```suggestion
-   In addition to the general rate limit on the Cloud Foundry API, requests for certain API endpoints related to services face a separate limit on concurrent requests. The Cloud Foundry API responds with HTTP status code `429` if a rate limit is reached and provides a Retry-After header suggesting when the client can attempt a retry. For more information, see https://docs.cloudfoundry.org/running/rate-limit-cloud-controller-api.html#Rate%20Limit%20Responses:%20Service%20Brokers .
```

**Review by sdanter — CHANGES_REQUESTED** [review]
Some small suggestions to maintain consistency regarding external links.

**Comment by sdanter** [comment]
Thank you for your contribution @FloThinksPi, the documentation will be updated with the next release.

**Pull request closed without merging**